src/main/java/school/faang/bjs2_81193/Army.java

```diff
@@ -0,0 +1,24 @@
+package school.faang.bjs2_81193;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Army {
+    private final List<Squad> armySquads = new ArrayList<>();
+
+    public void addSquad(Squad squad) {
+        armySquads.add(squad);
+    }
+
+    public int calculateTotalPower() throws InterruptedException {
+        AtomicInteger totalArmyPower = new AtomicInteger();
+        for (Squad squad : armySquads) {
```

Comment:
```suggestion
        List<Thread> threads = new ArrayList<>();

for (Squad squad : armySquads) {
    Thread thread = new Thread(() -> totalArmyPower.addAndGet(squad.calculateSquadPower()));
    thread.start(); // запустили поток и продолжаем цикл
    threads.add(thread);
}

for (Thread thread : threads) {
    thread.join();
}
```

при твоей реализации у тебя в цикле ты ждешь завершения потока в рамках цикла, то есть многопоточности совсем не будет, поток запустился, ждем его завершения, переходим к следующей итерации